rubyUtils.gradle

```diff
@@ -81,7 +81,7 @@ void gem(File projectDir, File buildDir, String gem, String version, String path
         jruby.runScriptlet("""
                 require 'rubygems/commands/install_command'
                 cmd = Gem::Commands::InstallCommand.new
-                cmd.handle_options ['--no-ri', '--no-rdoc', '${gem}', '-v', '${version}', '-i', '${path}']
+                cmd.handle_options ['--no-document', '${gem}', '-v', '${version}', '-i', '${path}']
```

Comment:
the end of an era :)

Reply:
`--no-rdoc` is dead long live `no-document` :new_moon_with_face: